staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go

```diff
@@ -240,6 +272,15 @@ type ValidatingAdmissionPolicyBindingSpec struct {
 	// Note that this is differs from ValidatingAdmissionPolicy matchConstraints, where resourceRules are required.
 	// +optional
 	MatchResources *MatchResources `json:"matchResources,omitempty" protobuf:"bytes,3,rep,name=matchResources"`
+
+	// ValidationActions declares how Validations of the referenced ValidatingAdmissionPolicy are enforced.
+	// Validations that evaluate to false are enforced according to ValidationActions.
+	// Failures defined by the ValidatingAdmissionPolicy's FailurePolicy are enforced according
+	// to ValidationActions only if the FailurePolicy is set to Fail, otherwise the failures are ignored. This
+	// includes compilation errors, runtime errors and misconfigurations of the policy.
+	// Required.
```

Comment:
check if API docs include the possible values.. might depend on whether the enum feature is on (which it is by default, but not in our swagger.json generation)

Reply:
explain what it means to include multiple things in the list, if order matters, if there's precedence, etc